<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>500</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MiniLockSync</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/images/MiniLock_15x15.png</normaloff>:/icons/images/MiniLock_15x15.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Logo</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="Home">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextBrowser" name="textBrowser_5">
             <property name="styleSheet">
              <string notr="true">border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
     font: 14px;
     min-width: 10em;
     show-decoration-selected: 1;</string>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Willkommen blabla, ...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="CloudService">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextBrowser" name="textBrowser">
             <property name="styleSheet">
              <string notr="true">border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
     font: 14px;
     min-width: 10em;
     show-decoration-selected: 1;</string>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Cloud-Service&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;if you want to encypt or decrypt your data, please use the second interface!&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;greez MLS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="ManageCloud">
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTreeView" name="treeView_fileBrowser">
             <property name="styleSheet">
              <string notr="true">QTreeView#treeView_fileBrowser {
	 border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
     font: 14px;
     min-width: 10em;
     show-decoration-selected: 1;
}

</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButton_addDir">
                 <property name="styleSheet">
                  <string notr="true">QPushButton#pushButton_addDir {
      background-color: #D3D3D3;
     border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
    font: bold 11px;
     min-width: 5em;
     padding: 6px;
 }
 
QPushButton#pushButton_addDir:hover {
	 background-color: #B0C4DE;
}

QPushButton#pushButton_addDir:focus {
	background-color:  #778899;
}</string>
                 </property>
                 <property name="text">
                  <string>Add</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_deleteDir">
                 <property name="styleSheet">
                  <string notr="true">QPushButton#pushButton_deleteDir {
     background-color: #D3D3D3;
     border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
    font: bold 11px;
     min-width: 5em;
     padding: 6px;
 }

QPushButton#pushButton_deleteDir:hover {
	 background-color: #B0C4DE;
}
QPushButton#pushButton_deleteDir:focus {
	background-color:  #778899;
}</string>
                 </property>
                 <property name="text">
                  <string>Delete</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_confirm">
                 <property name="styleSheet">
                  <string notr="true">QPushButton#pushButton_confirm {
     background-color: #D3D3D3;
     border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
    font: bold 11px;
     min-width: 5em;
     padding: 6px;
 }

QPushButton#pushButton_confirm:hover {
	 background-color: #B0C4DE;
}
QPushButton#pushButton_confirm:focus {
	background-color:  #778899;
}</string>
                 </property>
                 <property name="text">
                  <string>Confirm</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_16">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QTableWidget" name="tableWidget">
                 <property name="styleSheet">
                  <string notr="true">QTableWidget#tableWidget {
	 border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
     font: 14px;
     min-width: 10em;
     show-decoration-selected: 1;
}

</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="ManageWorkSaveDir">
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTreeView" name="treeView_dirBrowser">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>174</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QTreeView#treeView_dirBrowser {
	border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
     font: 14px;
     min-width: 10em;
     show-decoration-selected: 1;
}
</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButton_addWorkDir">
                 <property name="styleSheet">
                  <string notr="true">QPushButton#pushButton_addWorkDir {
      background-color: #D3D3D3;
     border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
    font: bold 11px;
     min-width: 8em;
     padding: 6px;
 }

QPushButton#pushButton_addWorkDir:hover {
	 background-color: #B0C4DE;
}
QPushButton#pushButton_addWorkDir:focus {
	background-color:  #778899;
}</string>
                 </property>
                 <property name="text">
                  <string>Add work directory</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_deleteDir_2">
                 <property name="styleSheet">
                  <string notr="true">QPushButton#pushButton_deleteDir_2 {
     background-color: #D3D3D3;
     border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
    font: bold 11px;
     min-width: 8em;
     padding: 6px;
 }
 
QPushButton#pushButton_deleteDir_2:hover {
	background-color: #B0C4DE;
}
QPushButton#pushButton_deleteDir_2:focus {
	background-color:  #778899;
}</string>
                 </property>
                 <property name="text">
                  <string>Delete work dir</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QTableWidget" name="tableWidget_dir">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>65</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QTableWidget#tableWidget_dir {
	 border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
     font: 14px;
     min-width: 10em;
     show-decoration-selected: 1;
}

</string>
                 </property>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>0</number>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButton_AddCloud">
                 <property name="styleSheet">
                  <string notr="true">QPushButton#pushButton_AddCloud {
     background-color: #D3D3D3;
     border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
    font: bold 11px;
     min-width: 8.4em;
     padding: 6px;
 }
 
QPushButton#pushButton_AddCloud:hover {
	background-color: #B0C4DE;
}
QPushButton#pushButton_AddCloud:focus {
	background-color:  #778899;
}</string>
                 </property>
                 <property name="text">
                  <string>Add cloud</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_delete_cloud">
                 <property name="styleSheet">
                  <string notr="true">QPushButton#pushButton_delete_cloud {
     background-color: #D3D3D3;
     border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
    font: bold 11px;
     min-width: 8.4em;
     padding: 6px;
 }

QPushButton#pushButton_delete_cloud:hover {
	background-color: #B0C4DE;
}
QPushButton#pushButton_delete_cloud:focus {
	background-color:  #778899;
}</string>
                 </property>
                 <property name="text">
                  <string>Delete Cloud</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <widget class="QTableWidget" name="tableWidget_cloud">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>65</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QTableWidget#tableWidget_cloud {
	 border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
     font: 14px;
     min-width: 10em;
     show-decoration-selected: 1;
}

</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
          <zorder>treeView_dirBrowser</zorder>
          <zorder></zorder>
          <zorder></zorder>
         </widget>
         <widget class="QWidget" name="CPULimitation">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="ChangePasword">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushHome">
         <property name="styleSheet">
          <string notr="true">QPushButton#pushHome {
     background-color: #D3D3D3;
     border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
     font: bold 11px;
     min-width: 5em;
     padding: 6px;
 }

QPushButton#pushHome:hover {
	 background-color: #B0C4DE;
}
QPushButton#pushHome:focus {
	background-color:  #778899;
}
</string>
         </property>
         <property name="text">
          <string>Welcome</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/buttons/buttons/HomeButtonRounded400.png</normaloff>:/buttons/buttons/HomeButtonRounded400.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushCloudService">
         <property name="styleSheet">
          <string notr="true">QPushButton#pushCloudService {
     background-color: #D3D3D3;
     border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
    font: bold 11px;
     min-width: 5em;
     padding: 6px;
 }

QPushButton#pushCloudService:hover {
	  background-color: #B0C4DE;
}
QPushButton#pushCloudService:focus {
	background-color:  #778899;
}</string>
         </property>
         <property name="text">
          <string>Work Dir</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/buttons/buttons/CloudService400x225.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushManageCloud">
         <property name="styleSheet">
          <string notr="true">QPushButton#pushManageCloud {
      background-color: #D3D3D3;
     border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
    font: bold 11px;
     min-width: 5em;
     padding: 6px;
 }

QPushButton#pushManageCloud:hover {
	background-color: #B0C4DE;
}
QPushButton#pushManageCloud:focus {
	background-color:  #778899;
}</string>
         </property>
         <property name="text">
          <string>Folders</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/buttons/buttons/Cloud-Management-Icon-635.jpg</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushManageSaveDir">
         <property name="styleSheet">
          <string notr="true">QPushButton#pushManageSaveDir {
     background-color: #D3D3D3;
     border-style: outset;
     border-width: 2px;
     border-radius: 6px;
     border-color: #708090;
    font: bold 11px;
     min-width: 5em;
     padding: 6px;
 }

QPushButton#pushManageSaveDir:hover {
	background-color: #B0C4DE;
}
QPushButton#pushManageSaveDir:focus {
	background-color:  #778899;
}</string>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/buttons/buttons/db-backup.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
